packages/connectkit/src/components/DaimoPayButton/index.tsx

```diff
@@ -67,30 +66,24 @@ const DaimoPayButtonRenderer: React.FC<DaimoPayButtonRendererProps> = ({
 }) => {
   const isMounted = useIsMounted();
   const context = useContext();
-  const { open, setOpen } = useModal();
```

Comment:
i think this fixes a bug? duplicate open/setOpen from the one in context

Reply:
this only affects `DaimoPayButton.Custom` tho which nobody is currently using, as far as I know.